pkg/component/etcd/etcd/crd.go

```diff
@@ -20,18 +21,20 @@ import (
 	"github.com/gardener/gardener/pkg/component"
 	"github.com/gardener/gardener/pkg/utils/flow"
 	gardenerutils "github.com/gardener/gardener/pkg/utils/gardener"
+	kubernetesutils "github.com/gardener/gardener/pkg/utils/kubernetes"
 )
 
 var (
 	//go:embed crds/templates/crd-druid.gardener.cloud_etcds.yaml
-	// CRD holds the etcd custom resource definition template
-	CRD string
+	// CrdCloudEtcds holds the etcd custom resource definition template
+	CrdCloudEtcds string
 	//go:embed crds/templates/crd-druid.gardener.cloud_etcdcopybackupstasks.yaml
-	etcdCopyBackupsTaskCRD string
+	crdEtcdCopyBackupsTasks string
 
 	etcdCRDName                = "etcds.druid.gardener.cloud"
 	etcdCopyBackupsTaskCRDName = "etcdcopybackupstasks.druid.gardener.cloud"
-	crdResources               []string
+	// resourceObjectKeyMap maps the ObjectKey of the CRD to its corresponding manifest
+	resourceObjectKeyMap map[client.ObjectKey]string
```

Comment:
```suggestion
	crdNameToResource map[string]string
```

For remaining occurrences as well.